sklearn/calibration.py

```diff
@@ -172,6 +174,7 @@ def fit(self, X, y, sample_weight=None):
                 warnings.warn("%s does not support sample_weight. Samples"
                               " weights are only used for the calibration"
                               " itself." % estimator_name)
+                sample_weight = check_array(sample_weight, ensure_2d=False)
```

Comment:
Uhm how it can be a bug since that `sample_weight` will not be given to the underlying estimator. Do I miss something?

Reply:
We sill need to pass sample_weight to _CalibratedClassifier